src/app/modals/examples/about-modal-example.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { TabsModule } from 'ngx-bootstrap/tabs';
```

Comment:
We're not using tabs to show multiple examples -- please remove

Reply:
We need TabsModule for to show API, html, Typescript docs.

Reply:
Right. In that case, you'll need to add TabsConfig as well.